ENG-2647 upgrade to jhipster 7.2

alepintus opened the pull request.

avdev4j commented:
@alepintus did you checked the templates too?

avdev4j commented:
I'm a bit surprised we can remove all the partials but it seems we are less linked to them. I remember we had to keep them (even without modifications) because of the different includes.

If removing them doesn't affect the generation, it's a good point.

alepintus commented:
Hi @avdev4j thanks for the review. 
About the inclusions, I checked our code and only 2 files are needed, so I removed the others: 
1. `field_validators.ejs` (was already present)
2. `relationship_validators.ejs` (I added this file in this PR)
